src/main/java/banking/account/view/BankView.java

```diff
@@ -0,0 +1,146 @@
+package banking.account.view;
+
+
+import banking.account.controller.BankController;
+import banking.account.domain.BasicAccount;
+import banking.account.dto.AccountDTO;
+import banking.account.exception.FormatException;
+
+import java.io.IOException;
+import java.math.BigDecimal;
+import java.text.DecimalFormat;
+import java.util.Scanner;
+
+import static banking.account.constant.ConstantMessage.*;
+
+public class BankView {
+    private static final Scanner sc = new Scanner(System.in);
+    private BankController bankController = BankController.getInstance();
+
+
+    public void startMenu () {
+
+        while(true) {
+            System.out.printf(CHOICE + "\n" + MENU + "\n");
+            String switchNum = input();
+
+            switch(switchNum) {
+                case "1":
+                    accountInput();
+                    break;
+                case "2":
+                    // 입금
+                    deposit();
+                    break;
+                case "3":
+                    // 출금
+                    withdraw();
+                    break;
+                case "4":
+                    // 송금
+                    trasfer();
+                    break;
+                case "5":
+                    // 계좌 조회
+                    findAccount();
+                    break;
+                case "6" :
+                    System.exit(0);
+                    break;
+                default:
+                    break;
+            }
+        }
+    }
+
+    private void trasfer() {
+        System.out.println(DEPOSIT + ACCOUNT_NUMBER );
+        String depositAccountNum = getFormat();
```

Comment:
변수명을 지을때 줄임말을 사용하는 것은 좋지 않습니다.
```suggestion
        String depositAccountNumber = getFormat();
```